webapp/src/ts/components/filters/form-type-filter/form-type-filter.component.ts

```diff
@@ -1,49 +1,30 @@
-import {
-  Component,
-  EventEmitter,
-  OnDestroy,
-  Output,
-  ViewChild,
-  Input,
-  OnInit
-} from '@angular/core';
+import { Component, EventEmitter, OnDestroy, Output, Input, OnInit } from '@angular/core';
 import { Store } from '@ngrx/store';
 import { sortBy as _sortBy } from 'lodash-es';
 import { Subscription } from 'rxjs';
 
 import { Selectors } from '@mm-selectors/index';
 import { GlobalActions } from '@mm-actions/global';
-import {
-  MultiDropdownFilterComponent,
-  MultiDropdownFilter,
-} from '@mm-components/filters/multi-dropdown-filter/multi-dropdown-filter.component';
-import { AbstractFilter } from '@mm-components/filters/abstract-filter';
-import { InlineFilter } from '@mm-components/filters/inline-filter';
+import { Filter } from '@mm-components/filters/filter';
 
 @Component({
   selector: 'mm-form-type-filter',
   templateUrl: './form-type-filter.component.html'
 })
-export class FormTypeFilterComponent implements OnDestroy, OnInit, AbstractFilter {
-  private globalActions;
+export class FormTypeFilterComponent implements OnDestroy, OnInit {
+  private globalActions: GlobalActions;
   private formsSubscription;
   forms;
   subscriptions: Subscription = new Subscription();
-  inlineFilter: InlineFilter;
```

Comment:
Do we need `inlineFilterClearSpy` in `webapp/tests/karma/ts/components/filters/status-filter.component.spec.ts` and `webapp/tests/karma/ts/components/filters/form-type-filter.component.spec.ts` I see you removed the references to `inlineFilter`

Reply:
Yes, we need it. Before, we used to have 2 classes, 1 for the old filter, and the other one for the new filter (inlineFilter).
Since I removed the old one, I have renamed `inline filter` to just `filter`. But forgot to rename those variables.

I've done the rename, thanks for catching that!